### DISCUSSION TITLE
Using zero-md inside a dynamic menu possible?

### DISCUSSION THREAD

**Discussion by mi-hol** [zerodevx/zero-md#99] [Q&A] [answered]
I try to enable markdown content  on a site currently using only HTML.

Website is https://mi-hol.github.io/Engenia/#
source code is https://github.com/mi-hol/Engenia/edit/main/index.html
relevant code from line #1050

The original html file is referenced by:
`menu1.addItem("Alt. Parts 0", "./01_General/05_AltParts/AltParts.html"); `

I want to replace it with https://github.com/mi-hol/Engenia/blob/main/01_General/05_AltParts/AltParts.md but all my attempts failed so far.

I've tried below syntax:

```
menu1.addItem("Alt. Parts 2", <zero-md src="https://github.com/mi-hol/Engenia/blob/main/01_General/05_AltParts/AltParts.md"</zero-md>); 
   
menu1.addItem("Alt. Parts 3", "<zero-md src='https://github.com/mi-hol/Engenia/blob/main/01_General/05_AltParts/AltParts.md'</zero-md>"); 
   
menu1.addItem("Alt. Parts 4", "<zero-md src="./01_General/05_AltParts/AltParts.md"</zero-md>"); 
```
   

but get errors:

Engenia/:1066  Uncaught SyntaxError: Unexpected token '<' (at Engenia/:1066:35)

Engenia/:960  Uncaught ReferenceError: menu1 is not defined
    at init (Engenia/:960:9)
    at onload (VM849 :1:1)

Any hint how to correct?

**Comment by zerodevx** [comment, marked as answer]
> The original html file is referenced by:
menu1.addItem("Alt. Parts 0", "./01_General/05_AltParts/AltParts.html"); 

Leave this part as it is, then replace [AltParts.html](https://github.com/mi-hol/Engenia/blob/main/01_General/05_AltParts/AltParts.html) with:

```html
<HTML>
<HEAD>
<TITLE>Links to Alternative Parts</TITLE>
<link rel="shortcut icon" href="../../EngeniaIcon.ico">

<script
  type="module"
  src="https://cdn.jsdelivr.net/gh/zerodevx/zero-md@2/dist/zero-md.min.js"
></script>

</HEAD>
<BODY BGCOLOR=FFFFFF TEXT=000000 LINK=0000FF VLINK=800080>

<zero-md src="./AltParts.md" />

</BODY>
</HTML>
```

**Reply by mi-hol** [reply]
Thank you so much for your kind assistance!
